code/espurna/config/hardware.h

```diff
@@ -4212,6 +4212,27 @@
     #define SENSOR_POWER_UNITS          POWER_WATTS
 
 // -----------------------------------------------------------------------------
+// KINGART_CURTAIN_SWITCH
+// -----------------------------------------------------------------------------
+#elif defined(KINGART_CURTAIN_SWITCH)
+
+    // Info
+    #define MANUFACTURER            "KingArt"
+    #define DEVICE                  "KINGART_CURTAIN_SWITCH"
```

Comment:
```cpp
    #define MANUFACTURER            "KINGART"
    #define DEVICE                  "CURTAIN_SWITCH"
```
We try to match the define above with these concatenated.

Reply:
Changed